app/index.js

```diff
@@ -1,5 +1,6 @@
 var Promise = require('bluebird'),
   child_process = require('child_process'),
+  fs = require('fs'),
```

Comment:
This shouldn't be needed once we use this.fs everywhere we can (hopefully that is everywhere).